### ISSUE TITLE
Python language server crashed by Libzip on 2021.6.1

### ISSUE THREAD

**Issue by FireBrother** [microsoft/pylance-release#1421]
## Environment data

-   Language Server version: 2021.6.1
-   OS and version: macOS Catalina 10.15.7
-   Python version (& distribution if applicable, e.g. Anaconda): Miniconda Python 3.7.10 64-bit

## Expected behaviour

It's expected to be fixed on 2021.6.1.

## Actual behaviour

Still facing the same issue as #1417 with version of 6.1.

## Logs

```
[Info  - 10:53:33 AM] Pylance language server 2021.6.1 (pyright e3f7cccc) starting
[Info  - 10:53:33 AM] Server root directory: /~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist
Libzip Error: Not a zip archive
    at makeLibzipError (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ZipFS.js:149:29)
    at new m (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ZipFS.js:121:28)
    at b.getZipSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ZipOpenFS.js:784:28)
    at b.makeCallSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ZipOpenFS.js:665:21)
    at b.readdirSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ZipOpenFS.js:582:21)
    at c.readdirSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ProxiedFS.js:191:28)
    at C.readdirSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/node_modules/@yarnpkg/fslib/lib/ProxiedFS.js:191:28)
    at I.readdirEntriesSync (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/src/common/fileSystem.ts:293:23)
    at t.PyrightFileSystem.processPartialStubPackages (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/src/pyrightFileSystem.ts:182:46)
    at _0x305a7e.ensurePartialStubPackages (/~/.vscode-server/extensions/ms-python.vscode-pylance-2021.6.1/dist/pyright-internal/src/analyzer/importResolver.ts:589:25)
```

## Code Snippet / Additional information

```python
XXX
```

**Comment by jakebailey** [comment]
Thanks for reporting; it seems like there are a few more cases for me to look at here.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2021.6.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202162-16-june-2021

**Comment by FireBrother** [comment]
Thanks for fixing
